mlir/python/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Copyright (c) 2025.
+
+[project]
+name = "mlir-python-bindings"
+version = "0.0.1"
```

Comment:
OOC, do you have any idea when we'd bump this? E.g., every new feature bump? (is there an option to just have sha attached or something, I see there is some support for dynamic version)